okta/resource_policy_rule_idp_discovery.go

```diff
@@ -242,7 +243,7 @@ func flattenPlatformInclude(platform *sdk.IdpDiscoveryRulePlatform) *schema.Set
 }
 
 func flattenAppInclude(app *sdk.IdpDiscoveryRuleApp) *schema.Set {
-	var flattened []interface{}
+	flattened := []interface{}{}
```

Comment:
This change from nil to empty slice was pointless. Both ways work for our purposes but it was changed while trying to track down the perpetual diff.